Utilities/NPC.gd

```diff
@@ -16,7 +18,6 @@ signal step
 
 func _ready():
 	$Position2D/Sprite.texture = texture
-	set_idle_frame(facing)
```

Comment:
If you remove this line, the NPC's will default to facing down, instead of up. Please fix this by either changing the default facing to be down, or editing the NPC's orientation on the maps to the correct one.